_ont/ont-huawei-ma5671a-root.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: Root Procedure for Huawei MA5671A
+title: Root Procedure for Huawei MA5671A (V2)
 has_children: false
 parent: Huawei MA5671A
 layout: default
+nav_exclude: true
```

Comment:
Doesn't this hide the page from the menu? is it wanted?

Reply:
yes my idea is to have the url `ont-huawei-ma5671a-root` with the new web verisone, the url `ont-huawei-ma5671a-list-root` with the list of old versions for documentation and not visible from the menu, but accessible from a link. This, however, I would like to do on a separate PR, perhaps also restoring V1.